github_tutorial.md

```diff
@@ -46,12 +49,12 @@ Wanneer je op je eigen laptop/pc werkt aan bestanden van een GitHub repository,
 Wanneer je wijzigingen hebt doorgevoerd op je eigen "fork" van een repository, maar deze wilt doorgeven aan de repository van VNG Realisatie, dan doe je een "pull request". Je verzoekt dan aan de beheerder van de repository om je voorgestelde wijzigingen/toevoegingen te accepteren en "naar binnen te trekken".
 
 **origin**
-Wanneer je op je eigen laptop/pc werkt aan bestanden, is "origin" de alias voor je eigen GitHub (remote) repository waarop je wijzigingen op kunt doorvoeren (via een `git push -u origin branchnaam`).
+Wanneer je op je eigen laptop/pc werkt aan bestanden, is "origin" de alias voor je eigen GitHub (remote) repository waarop je wijzigingen kunt doorvoeren (via een `git push -u origin branchnaam`).
 
 **upstream**
 Wanneer je op je eigen laptop/pc werkt aan bestanden, is "upstream" de alias voor de GitHub (remote) repository van VNG-Realisatie. Dit gebruik je wanneer je de lokale kopie van de repository wilt actualiseren met alle door anderen doorgevoerde wijzigingen (via een `git pull upstream master`).
 
-# GitHub workflow voor open Standaarden
+# GitHub workflow (in de browser) voor open Standaarden
```

Comment:
Deze toevoeging is denk ik niet correct. Het gaat om de workflow, niet om de tool (i.c. browser) die je daarvoor gebruikt.

Reply:
Bij nader inzien heb je denk ik wel gelijk. Er wordt hier alleen een workflow beschreven en er zitten geen platform afhankelijke beschrijvingen in. Deze workflow geldt voor de browser maar ook voor de desktop applicatie.

Reply:
Is aangepast.